README.md

```diff
@@ -425,6 +462,44 @@ If you're using a certificate signed by a private CA, you will also need to add
 
 If you're using a self-signed certificate, e.g. for testing or development purposes, you can set the `REJECT_UNAUTHORIZED` parameter to `false` in your `.env.local`. This will disable certificate validation, and allow Chat UI to connect to your custom endpoint.
 
+#### Specific Embedding Model
+
+A model can use any of the embedding models defined in `.env.local`, (currently used when web searching),
+by default it will use the first embedding model, but it can be changed with the field `embeddingModel`:
+
+```env
+TEXT_EMBEDDING_MODELS = `[
+  {
+    "name": "Xenova/gte-small",
+    "maxSequenceLength": 512,
+    "endpoints": [
+      {"type": "xenova"}
+    ]
+  },
+  {
```

Comment:
could you add [dummy authorization field ](https://github.com/mikelfried/chat-ui/blob/embedding_models/src/lib/server/embeddingEndpoints/tei/embeddingEndpoints.ts#L51-L52) example here?